cmake/TestSetup.cmake

```diff
@@ -11,6 +11,27 @@
 # the public, perform publicly and display publicly, and to permit others to do so.
 #=========================================================================================
 
+
+
+# Search for the python interpreter
+if ( CMAKE_VERSION VERSION_LESS "3.12.4")
```

Comment:
Make this 3.12 or 3.12.0 - you can only browse the documentation for the latest patch release of each minor version, but I don't think documentation-worthy changes are made in patch releases. So `FindPython` should be in 3.12.0, too.

Reply:
Well other than this, is it good to go?